frontend/public/components/events.jsx

```diff
@@ -516,7 +516,7 @@ class EventStream extends React.Component {
               className="co-sysevent-stream__timeline__btn"
             />
             <div className="co-sysevent-stream__timeline__end-message">
-              There are no events before <Timestamp timestamp={this.state.oldestTimestamp} />
+              Events older than 2 hours are not stored.
```

Comment:
The default TTL is one hour, but I believe it can be changed. We might not be able to hard-code the message.

Reply:
I just noticed [the empty state message is `No events in the past hour`](https://github.com/openshift/console/blob/385f62dac05fad1757825dbc197ff86b96ea5c8c/frontend/public/components/events.jsx#L251).  If the TTL can be changed, the empty state message also needs updating.  Perhaps we just do:

`Older events are not stored.` and `No events` so as to keep the messages precise?  @andybraren, your thoughts?

Reply:
I like those tweaks @rhamilto, both of which are better than what they are today. I posted in [#forum-monitoring](https://coreos.slack.com/archives/C0VMT03S5/p1607723565406400) to see if the 1 hour is configurable.

Reply:
Oh, I just found this which suggests the default is 3 hours and isn't configurable? 🤷‍♂️
https://access.redhat.com/solutions/5170631

Reply:
https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/v4.1.0/config/defaultconfig.yaml#L96